ag/task.go

```diff
@@ -22,3 +22,10 @@ func (t *Task) MarkDeleted() {
 func (t *Task) IsDeleted() bool {
 	return strings.HasPrefix(t.Title, deleted)
 }
+
+// LocalName returns the task name without the assignment part,
+// i.e. if a task has the name "assignment1/hello_world". it returns "hello_world".
+func (t *Task) LocalName() string {
+	s := strings.Split(t.GetName(), "/")
```

Comment:
Recommended approach with Go 1.18. You will need to also update `go.mod`
```go
	_, s, _ := strings.Cut(t.GetName(), "/")
	return s
```

Reply:
No longer valid